packages/react/src/components/Grid/Column.js

```diff
@@ -51,6 +51,7 @@ const spanPropType = FeatureFlags.enabled('enable-css-grid')
         start: PropTypes.number,
         end: PropTypes.number,
       }),
+      PropTypes.arrayOf(PropTypes.oneOf(['25%', '50%', '75%', '100%'])),
```

Comment:
Would something like this work:

```js
const percentSpanType = PropTypes.oneOf(['25%', '50%', '75%', '100%']);

//
? PropTypes.oneOfType([
    PropTypes.bool,
    PropTypes.number,
    PropTypes.shape({
      span: PropTypes.oneOfType([PropTypes.number, percentSpanType]),
      offset: PropTypes.number,
      start: PropTypes.number,
      end: PropTypes.number,
    }),
    percentSpanType
  ]);
```

Only reason why I'm curious is if we could support offsets or not with percent span 🤔

Reply:
@joshblack Great idea - I got it working! I added a new column to the `offset` story that uses percent-based span with offsets. Check it out and let me know what you think.